scripts/o3de/tests/test_repo.py

```diff
@@ -65,6 +65,130 @@
 }
 '''
 
+#This holds the repo 2.0.0 schema, which houses all relevant objects in file
+TEST_O3DE_REPOC_FILENAME = '3b14717bafd5a3bd768d3d0791a44998c3bd0fb2bfa1a7e2ee8bb1a39b04d631.json'
+TEST_O3DE_REPOC_JSON_PAYLOAD = '''
+{
+	"repo_name": "testgem3",
+
+	"origin": "Studios",
+
+    "$schemaVersion":"2.0.0",
+    
+	"repo_uri": "https://downloads.testgem3.com/o3de-repo",
+
+	"summary": "Studios Repository for the testgem3 Gem.",
+
+	"additional_info": "",
+
+	"last_updated": "2023-01-19",
+
+	"gems": [{
+		"gem_name": "testgem3",
+		"display_name": "testgem3 2",
+		"download_api": "HTTP",
+		"license": "Community",
+		"license_url": "https://www.testgem3.com/testgem3-community-license",
+		"origin": "Persistant Studios - testgem3.com",
+		"requirements": "Users will need to download testgem3 Editor from the <a href='https://www.testgem3.com/download/'>testgem3 Web Site</a> to edit/author effects.",
+		"documentation_url": "https://www.testgem3.com/docs/testgem3-v2/plugins/o3de-gem/",
+		"type": "Code",
+		"summary": "The testgem3 Gem provides real-time FX solution for particle effects.",
+		"canonical_tags": [
+			"Gem"
+		],
+		"user_tags": [
+			"Particles",
+			"Simulation",
+			"SDK"
+		],
+		"dependencies": [],
+		"objects": [{
```

Comment:
I wonder if a better name would be `object_versions`  or `version_data`

Reply:
This is like a manifest for content advertisement based on versions.